source/options-storage.ts

```diff
@@ -20,6 +20,7 @@ const defaults = Object.assign({
 const migrations = [
 	// Merged on July 2nd
 	featureWasRenamed('sticky-discussion-list-toolbar', 'sticky-conversation-list-toolbar'),
+	featureWasRenamed('sticky-conversation-sidebar', 'sticky-sidebar'),
```

Comment:
```suggestion
	featureWasRenamed('sticky-conversation-sidebar', 'sticky-sidebar'),  // Merged September 
```

Also you can remove the 2 lines above I left them so it can be used as a “live” example how to do migrations